front/app/templates/application.hbs

```diff
@@ -1,21 +1,25 @@
-<a href="https://github.com/dikaiosune/rust-dashboard" target="_blank"><img style="position: absolute; top: 0; left: 0; border: 0;" alt="Fork me on GitHub" src="https://s3.amazonaws.com/github/ribbons/forkme_left_darkblue_121621.png"></a>
+<a href="https://github.com/dikaiosune/rust-dashboard" target="_blank">
```

Comment:
Did you format this manually, or use an automated reformatter? Whatever engine is included with `atom-beautify` created these long-single-line monstrosities.